frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/portlets/licenses/LicensesPortlet.java

```diff
@@ -155,6 +197,81 @@ private void addLicenseBreadcrumb(RenderRequest request, RenderResponse response
     }
 
     @UsedAsLiferayAction
+    public void update(ActionRequest request, ActionResponse response) throws PortletException, IOException {
+        License license = new License();
+        LicenseService.Iface client = thriftClients.makeLicenseClient();
+        String licenseId = request.getParameter(LICENSE_ID);
+        User user = UserCacheHolder.getUserFromRequest(request);
+
+        if (!Strings.isNullOrEmpty(licenseId)) {
+            try {
+                license = client.getByID(licenseId, user.getDepartment());
+            } catch (TException e) {
+                log.error("Error updating license:", e);
+            }
+        }
+
+        license = updateLicenseFromRequest(license,request);
+
+        RequestStatus requestStatus=RequestStatus.FAILURE;
```

Comment:
It might be better to write `RequestStatus requestStatus;` and add `requestStatus = RequestStatus.FAILURE;` to the catch block below.